Makefile

```
@@ -4,10 +4,12 @@ VERSION := 0.5.4
 SERVICE_NAME := redis-operator
 
 # Docker image name for this project
-IMAGE_NAME := spotahome/$(SERVICE_NAME)
+INAME ?= spotahome
+IMAGE_NAME := $(INAME)/$(SERVICE_NAME)
```

Maybe, If we want to allow users to configure the image names, Why not letting them define the whole image name?

```suggestion
IMAGE_NAME ?= spotahome/$(SERVICE_NAME)
```

---

I've tried that first but since `SERVICE_NAME` is a constant variable adding your suggestion may raises problems on `make` calls if the provided service name won't match.

I'm thinking on adding more granularity when defining `REPOSITORY` variable, let me know your thoughts 😄